### PR TITLE
Reorganize navigation: move success views into path

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-wallet/src/ui/react/App.tsx [hunk 1]
```diff
@@ -22,6 +23,8 @@ const App = () => {
       <Route exact path="/terms" component={TermsAndConditionsScreen} />
       <Route exact path="/privacy" component={PrivacyPolicy} />
       <Route exact path="/create" component={CreateFlow} />
+      <Route exact path="/connectionSucceed" component={ConnectionSuccess} />
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/App.tsx]: We have a component named `ConnectionSuccess` and we are creating url named `connectionSucceed`. It hurts...  I see 2 directions: 
- Rename this path to `connectinSuccess` and then make PR with rename
- PR with rename ConnectionSuccess -> ConnectionSucceed and after that, we can merge this PR (1617)

Comment (xitronix) [reply universal-login-wallet/src/ui/react/App.tsx]: ✔️

File: universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx [hunk 2]
```diff
@@ -2,12 +2,12 @@ import React, {useState} from 'react';
 import {ConnectionFlow} from '@universal-login/react';
 import {ConnectSelector} from './ConnectSelector';
 import {useServices} from '../../hooks';
-import {ConnectionSuccess} from '../Modals/ModalTxnSuccess';
-
-export type ConnectModal = 'connectionFlow' | 'selector' | 'connectionSucceed';
+import {useHistory} from 'react-router';
 
+export type ConnectModal = 'connectionFlow' | 'selector';
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx]: add new line

Comment (xitronix) [reply universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx]: ✔️

Comment (xitronix) [reply universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx]: ✔️

**Pull request merged**